src/migrations/1741025559300_scene-admin.ts

```diff
@@ -1,24 +0,0 @@
-import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
```

Comment:
You seem to be removing a migration file that was probably deployed and ran. Have you rolled back this change in the environments this service was deployed?